games-fps/alephone/additional-files/startscript.sh.in

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+EXECUTABLE="`finddir B_SYSTEM_BIN_DIRECTORY`/alephone"
```

Comment:
Well, it isn't secondary arch friendly. Any idea what can i do here?

Reply:
I would split this recipe into 4 recipes:
* `alephone-20180414.recipe`
* `alephone_marathon-20150907.recipe`
* `alephone_marathon2-20150907.recipe`
* `alephone_marathoninfinity-20150907.recipe`

The first recipe would define `commandDinDir` (and `commandSuffix`) and provide `cmd:alephone$commandSuffix = $portVersion`.
The other 3 would have `ARCHITECTURES="any"` and would `REQUIRE` `cmd:alephone` (with no `$commandSuffix`). And we could insert the following bloc to each `alephone_marathon*`'s `INSTALL`:
```bash
	ALEPHONE="`finddir B_SYSTEM_BIN_DIRECTORY`/alephone
	DATA_PATH="`finddir B_SYSTEM_DATA_DIRECTORY`/AlephOne/Marathon"
	cat >"$appsDir"/Marathon <<EOF
#!/bin/sh
exec $ALEPHONE "$DATA_PATH"
EOF
	chmod +x "$appsDir"/Marathon
```
Of course, for `alephone_marathon2` and `alephone_marathoninfinity` we would replace `Marathon` by `Marathon 2` and `Marathon Inifinity` in the definition of `$DATA_PATH`.

And in case each of these three `alephone_marathon*` recipes does not depend on the other two, it gives the user more flexibility since the package need not be all installed.

BTW, you might also wish to switch to `.tar.gz` (instead of `.zip`) for the `alephone_marathon*` recipes.

Reply:
True, we don't want to have 100mb packages for every arch....